geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/ExplainCommand.scala

```diff
@@ -32,5 +34,7 @@ class ExplainCommand(parent: JCommander) extends CommandWithCatalog(parent) with
 
 object ExplainCommand {
   @Parameters(commandDescription = "Explain how a GeoMesa query will be executed")
-  class ExplainParameters extends RequiredCqlFilterParameters {}
+  class ExplainParameters extends GeoMesaConnectionParams
```

Comment:
I haven't looked closely, but the idea is that parameters get re-used all over, so having them as mixins maximizes re-use of the code. Previously we extended classes, but that didn't allow for mix-n-match.